AR Collaboration/Assets/Client/Scripts/AudioComponentDelete.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioComponentDelete : MonoBehaviour
+{
+    void Start()
+    {
+        string microphone = "Built-in Microphone";
+
+        foreach (var device in Microphone.devices)
+        {
+            print(device);
+            if (device.Contains("Webcam"))
+            {
+                print("Found microphone");
+                microphone = device;
+            }
+        }
+
+        AudioSource audioSource = GetComponent<AudioSource>();
+        print("Starting recording");
```

Comment:
Do we want to leave these as console logs, or do you want them to pop up in the users view?